servers/republik/graphql/resolvers/_mutations/updateMe.js

```diff
@@ -199,6 +199,9 @@ module.exports = async (_, args, context) => {
         )
       }
     }
+    if (phoneNumber) {
```

Comment:
`null` is used to unset, `undefined` to not update. Make sure to support `null`.

Reply:
Let's open a issue for that one. Also needs string length limits. But should not be part of this PR. Otherwise it will never get merged.